<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWindow</class>
 <widget class="QMainWindow" name="ChatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChatWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="3">
     <layout class="QVBoxLayout" name="rightPane" stretch="0,0,0,0,0,0,0">
      <item>
       <widget class="QTextBrowser" name="chatHistory"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="sendLayout">
        <item>
         <widget class="QLineEdit" name="sendText">
          <property name="placeholderText">
           <string>message</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sendButton">
          <property name="text">
           <string>Send</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="generalSettingsLabel">
        <property name="text">
         <string>General</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="generalSettingsLayout">
        <item>
         <widget class="QLineEdit" name="hostText">
          <property name="text">
           <string>localhost</string>
          </property>
          <property name="placeholderText">
           <string>host</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="portText">
          <property name="text">
           <string>2048</string>
          </property>
          <property name="placeholderText">
           <string>port</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clientConnectButton">
          <property name="text">
           <string>Connect as client</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="serverConnectButton">
          <property name="text">
           <string>Start Server</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="disconnectButton">
          <property name="text">
           <string>Disconnect</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="networkRadioLabel">
        <property name="text">
         <string>Network interface</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="networkRadioLayout">
        <item>
         <widget class="QRadioButton" name="unixApiRadio">
          <property name="text">
           <string>Direct Unix API</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="knApiRadio">
          <property name="text">
           <string>KN API</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="0" rowspan="2">
     <layout class="QVBoxLayout" name="leftPane">
      <item>
       <widget class="QLabel" name="peersLabel">
        <property name="text">
         <string>Peers</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="peerList"/>
      </item>
      <item>
       <widget class="QLineEdit" name="nameEdit">
        <property name="placeholderText">
         <string>name</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>613</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
